repour/adjust/pme_provider.py

```diff
@@ -80,8 +80,17 @@ def get_extra_parameters(extra_adjust_parameters):
     @asyncio.coroutine
     def adjust(repo_dir, extra_adjust_parameters, adjust_result):
         nonlocal execution_name
+
+        temp_build_parameters = []
+        if timestamp:
+            temp_build_parameters.append("-DversionIncrementalSuffix=" + timestamp + "-redhat")
```

Comment:
Will this override the default versionIncrementalSuffix value?

Reply:
This option is listed again after the default one. So I believe it will override the default one.